static/js/settings_bots.js

```diff
@@ -444,13 +452,9 @@ exports.set_up = function () {
     });
 
     $("#active_bots_list").on("click", "a.download_bot_zuliprc", function () {
-        var bot_info = $(this).closest(".bot-information-box");
-        var email = bot_info.find(".email .value").text();
-        var api_key = bot_info.find(".api_key .api-key-value-and-button .value").text();
-
-        $(this).attr("href", exports.generate_zuliprc_uri(
-            $.trim(email), $.trim(api_key)
-        ));
+        var bot_info = $(this).closest(".bot-information-box").find(".bot_info");
+        var bot_id = bot_info.data("user-id");
```

Comment:
Stylically, we prefex `.attr("data-user-id")`, basically because storing using `.data()` in jQuery is magic and can store things in different places.